.github/workflows/publish-packages.yml

```
@@ -69,38 +64,82 @@ jobs:
+          echo "LDFLAGS=-mmacosx-version-min=11.0" >> $GITHUB_ENV
+
+      - name: Force vendored OpenSSL for macOS
+        if: startsWith(matrix.build.OS, 'macos')
+        shell: bash
+        run: |
+          echo "OPENSSL_STATIC=1" >> $GITHUB_ENV
+          echo "OPENSSL_VENDORED=1" >> $GITHUB_ENV
 
-      - name: Build (linux/macos/windows)
+      - name: Build (linux/macos)
         uses: actions-rs/cargo@v1
```

actionlint (1.7.9)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)